ansible_collections/arista/avd/plugins/action/eos_designs_documentation.py

```diff
@@ -121,8 +123,19 @@ def main(self, validated_args: dict, task_vars: dict, result: dict) -> dict:
 
         if output.digital_twin:
             content = {str(key).replace("_", "-"): value for key, value in asdict(output.digital_twin).items() if value is not None}
+            # Adjust formatting of the dumped Digital Twin artifact
+            match output.digital_twin:
+                case ACTDigitalTwin():
+                    serialized_content = "---\n" + sub(
+                        r"^(\s*-|\s{4})",
+                        lambda match: (" " * 4 + text) if (text := match.group(0)).startswith("  -") else (" " * 2 + text),
+                        yaml.dump(content, Dumper=AnsibleDumper, sort_keys=False, indent=2, width=130),
+                        flags=MULTILINE,
+                    )
```

Comment:
Consider adding kwargs for clarity:
```suggestion
                    serialized_content = "---\n" + sub(
                        pattern=r"^(\s*-|\s{4})",
                        repl=lambda match: (" " * 4 + text) if (text := match.group(0)).startswith("  -") else (" " * 2 + text),
                        string=base_yaml,
                        flags=MULTILINE,
                    )
```

Reply:
please use f-strings for compiling strings instead of +

Reply:
Also the sub stuff should preferably be done above instead of inline in the string compilation. (for clarity)

Reply:
All items should be fixed now